loggerr.d.ts

```diff
@@ -0,0 +1,91 @@
+
+declare module 'loggerr' {
+  type Formatter = 'browser' | 'bunyan' | 'cli' | 'default';
```

Comment:
I recently added a new `cli-simple` https://github.com/wesleytodd/loggerr/blob/main/formatters/cli-simple.js

Reply:
Oh just seeing this one! https://github.com/wesleytodd/loggerr/pull/17